monai/data/meta_tensor.py

```diff
@@ -89,6 +90,11 @@ def __new__(cls, x, affine: torch.Tensor | None = None, meta: dict | None = None
             out.affine = out.get_default_affine()
         out.affine = out.affine.to(out.device)
 
+        # if we are creating a new MetaTensor, then deep copy attributes
+        if isinstance(x, torch.Tensor) and not isinstance(x, MetaTensor):
+            out.meta = deepcopy(out.meta)
+        out.affine = out.affine.to(out.device)
```

Comment:
Hi @wyli ,

Could you please help confirm whether this part logic works well with your PR design https://github.com/Project-MONAI/MONAI/pull/4131?

Thanks in advance.

Reply:
yes, this makes sense, perhaps we need another method such as `as_meta_tensor` to provide an option of avoiding heavy deepcopy...

Reply:
I don't think the deepcopy will be heavy. It's only the meta data (not the image) and would only happen when constructing a `MetaTensor` from `torch.Tensor`, which we wouldn't be doing often.

Reply:
not sure if we should have these two assumptions... 1. metadata is always lightweight 2. we don't often convert torch tensor to meta tensor. It's ok for the current use cases, but we should also think about future extensions.